packages/hint/docs/user-guide/telemetry/index.md

```diff
@@ -0,0 +1,33 @@
+# Telemetry
+
+## What are we tracking
+
+The first time you run `webhint` we will ask you if you want to send us
```

Comment:
And what do you propose to activate the telemetry or not?

Reply:
> And what do you propose to activate the telemetry or not?

We should not collect data without the user consent. 

---

Maybe ask the question after the first results are in? Then store the result somewhere independent of the package (e.g.: in `~/.hint/`) so that we will not need to ask again on future installs (e.g.: when the user uses `webhint` using `npx` multiple times)?

@webhintio/core Thoughts?

Reply:
> We should not collect data without the user consent.

That is why we are asking :P

The answer is already store in the user data folder, so no need to do anything about that.

Reply:
> The answer is already store in the user data folder, so no need to do anything about that.

Yes, sorry I missed/forgot about the `configstore` part.

Then maybe move the question after the first results are in?

Reply:
My concern about asking people after the results is they could think we are going to send them and we don't want them to think that because they will deny it most likely.

I'm not sure what to do about this. Other projects ask initially and websites is what they do. Is there any article out there about best practices?

Reply:
> My concern about asking people after the results is they could think we are going to send them and we don't want them to think that because they will deny it most likely.

I think if we have a short message making it clear before the question, the odds improve.

> Other projects ask initially and websites is what they do. 

I opinion is that users are more willing to accept it once they see the value of the tool, rather than before.

> Is there any article out there about best practices?

I haven't seen one, but usually [asking at the right time (in the right context) and providing information seems to yield the best result](https://www.youtube.com/watch?v=2u5qR3trwEY).

Reply:
Maybe add a screenshot? Maybe use the same package we are using for the messages surrounded on a square?